cookiecutter.json

```diff
@@ -4,6 +4,7 @@
   "css_addon": ["Bootstrap", "Tailwind", "None"],
   "use_logrus": ["yes", "no"],
   "use_heroku": ["yes", "no"],
+  "use_mock_server": ["yes", "no"],
```

Comment:
Should we always provide a mock server for API variant if we're always using OpenAPI for API variant? Could there be a situation where the mock server isn't necessary?

Reply:
Yes, there are cases when a mock server is not compulsory (so a nice-to-have). See: https://nimblehq.co/compass/development/testing/api-mocking/#1-api-and-client-applications-are-developed-concurrently

This could be challenged in the sense that for automated testing, we could still make use of a mock server 💡